backend/etcd/etcd.go

```diff
@@ -18,9 +18,13 @@ type Client struct {
 }
 
 func New(machines []string) (*Client, error) {
-	newClient, err := goetcd.New(goetcd.Config{
+	return NewV2Raw(goetcd.Config{
 		Endpoints: machines,
 	})
+}
+
+func NewV2Raw(config goetcd.Config) (*Client, error) {
```

Comment:
I think I'd rather use something like `FromConfig` as a suffix. Raw is very generic.